docs/content/en/docs/install/integrate.md

```diff
@@ -79,24 +79,24 @@ Otherwise, the deployment is kept in a pending state
 until the infrastructure is capable of accepting deployments again.
 
 A more comprehensive example can be found in our
-[examples folder](./examples/sample-app/),
+[examples folder](https://github.com/keptn/lifecycle-toolkit/tree/main/examples/sample-app),
```

Comment:
for the future, the source of truth for the examples is the examples repo that can be found here: https://github.com/keptn-sandbox/lifecycle-toolkit-examples
The examples folder in the KLT repo could be further ahead than the latest released KLT version and therefore not be compatible.

Reply:
👍